components/developer/jenkins/Makefile

```diff
@@ -0,0 +1,88 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016-2017 Jim Klimov
+#
+include ../../../make-rules/shared-macros.mk
+
+# NOTE: Before bumping to the new core version, take a look at
+#   https://jenkins.io/changelog/
+# for "weather report" (user votes on good/bad state of the weekly)
+# At some point it may make sense to deliver packaged LTS releases,
+# and the initial recipe supports that notion already just in case,
+# but no LTS issued so far includes updated libzfs.jar (v0.8+ needed).
+COMPONENT_NAME=		jenkins
+COMPONENT_MAJOR_VERSION=	2
+COMPONENT_MINOR_VERSION=	65
+COMPONENT_VERSION=	$(COMPONENT_MAJOR_VERSION).$(COMPONENT_MINOR_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).war
+COMPONENT_FMRI=		developer/jenkins-core-weekly
```

Comment:
Unfortunately no - some people apparently use the stale but predictable LTS releases :)
Regarding mediation - feel free to add the "developer/Jenkins", it would make sense perhaps. But note that one bulky matter is delivery of a WAR file, and another is of deploying and running it in some container - or standalone as done in this initial package. Other packages may follow if there's interest and a reliable idea how to implement it. Maybe just deliver and kick a service to re-deploy into a corresponding appserver like tomcat, tomcat8, glasshish/payara... this is not in scope for this PR anyhow.